infra/fastly/terraform/dl-sandbox.k8s.dev/vcl/binaries.vcl

```diff
@@ -44,8 +44,8 @@ sub vcl_fetch {
     set beresp.stale_while_revalidate = 60s; // 1 minute
   }
 
-  //Ensure version markers are not cached
-  if (req.url.ext == "txt") {
+  # Ensure HTML, JSON and TXT files are not cached at the edge
+  if (req.url.ext ~ "(html|json|txt)$") {
```

Comment:
ummm are we mutating these??

Reply:
are there some specific paths we should exclude instead?

Reply:
The version markers (latest|stable) are mutated.

exclude specific paths or subpaths for those kind of files is not worth it IMHO. They are too small to cache at the edge.

Reply:
Well, today, but we may trip over this assumption in the future ... someday we will add a huge json object to releases

(e.g. OCI image individual blobs)

I would strongly prefer to except the marker paths if we can instead.
We can add more exceptions if we need, but we don't want to over-except and miss something important.

Reply:
Yeah. We may encounter it in the future but it's unlikely to happen in the short term (EOY ?).

Reply:
The problem is who else will even know and expect this, and then later we figure out the issue by way of the budget exploding.

Reply:
Versus it will be very clear if we're caching something we didn't want cached, and that's a quick fix.

Reply:
I tried to be specific about the url paths for the version markers. PTAL.